legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -324,9 +329,11 @@ def party_role_revision_json(transaction_id, party_role_revision) -> dict:
         party_revision = VersionedBusinessDetailsService.get_party_revision(transaction_id, party_role_revision)
         party = {
             **VersionedBusinessDetailsService.party_revision_json(transaction_id, party_revision),
-            'appointmentDate': datetime.date(party_role_revision.appointment_date).isoformat(),
             'cessationDate': cessation_date,
```

Comment:
cessation date is also for a role

Reply:
so it should be part of roles object?

Reply:
for IA, correction etc it is part of roles object. For COD, AR with COD roles doesnt exist and all these attributes (role, appointment date and cessation date ) are in the party object